src/bp/nlu-core/entities/duckling-extractor/duckling-provider.ts

```diff
@@ -0,0 +1,71 @@
+import Axios, { AxiosInstance } from 'axios'
```

Comment:
All the content of this file comes from previous duckling extractor file, no need to review